ci/filter_example_setup.sh

```diff
@@ -12,6 +12,7 @@ ENVOY_FILTER_EXAMPLE_SRCDIR="${BUILD_DIR}/envoy-filter-example"
 ENVOY_FILTER_EXAMPLE_TESTS=(
     "//:echo2_integration_test"
     "//http-filter-example:http_filter_integration_test"
+    "//contrib/golang/filters/http/test/..."
```

Comment:
I think this will make contrib tests run as part of standard CI, which I think we want to avoid for now.   If I'm wrong please feel free to correct me otherwise let's back it out
/wait-any

Reply:
okay, reverted this line and merged main, PTAL again, thanks